commit 92929f9777b9c39a2829a9db1c19ef37859dc7fc
Author: Matthew Thode <mthode@mthode.org>
Date:   Sun Mar 15 14:00:14 2020 -0500

    add soname to shared libraries
    
    Found in Gentoo QA
    
    * QA Notice: The following shared libraries lack a SONAME
    * /usr/lib/libbemenu.so.0.4.0
    
    Signed-off-by: Matthew Thode <mthode@mthode.org>

diff --git a/GNUmakefile b/GNUmakefile
index c82bfd3..5b7b081 100644
--- a/GNUmakefile
+++ b/GNUmakefile
@@ -36,7 +36,7 @@ wayland: bemenu-renderer-wayland.so
 	$(LINK.c) -c $(filter %.c,$^) $(LDLIBS) -o $@
 
 $(libs): %: VERSION .git/index
-	$(LINK.c) -shared -fPIC $(filter %.c %.a,$^) $(LDLIBS) -o $(addsuffix .$(VERSION), $@)
+	$(LINK.c) -shared -fPIC $(filter %.c %.a,$^) $(LDLIBS) -o $(addsuffix .$(VERSION), $@) -Wl,-soname
 	ln -fs $(addsuffix .$(VERSION), $@) $(addsuffix .$(firstword $(subst ., ,$(VERSION))), $@)
 	ln -fs $(addsuffix .$(VERSION), $@) $@
 
